auacm/app/database.py

```diff
@@ -1,18 +1,17 @@
-'''Database handlers.'''
+'''Creates the database.'''
 from sqlalchemy import create_engine
 from sqlalchemy.orm import Session
 from sqlalchemy.ext.automap import automap_base
 
 # Create global database variables
-Base = automap_base()
-engine = create_engine('mysql+pymysql://acm@localhost/acm?charset=utf8')
-connection = engine.connect()
-session = Session(engine)
+DATABASE_BASE = automap_base()
```

Comment:
I think I'm ok with this, but I'd like a second opinion ( @mcprice30 @WilliamHester ).

My guess is that you uppercased these to satisfy pylint. But uppercasing like this implies that these values are constants, which they're not: they're global objects.

I know global objects are generally taboo for good reasons, but I'm not sure how we would reasonably go about avoiding them for at least some things like `app` and `session`. My only idea is a singleton that creates/returns the object on a getter method.

So I see three options, and I'm not sure which is best:
1. Keep it as is, and just know that the caps doesn't mean it's a constant
1. Disable pylint for these specific object via comments
1. Reorganize it so that we don't have explicitly global objects

Thoughts?